### ISSUE TITLE
[Android] I just downloaded last update to the browser on my phone and YouTube showing occasional ads on YouTube. Tried turning off shields and turning them back on. I'm using Alcatel V3 Android 9. I normally use monoarm64 for updating browser.

### ISSUE THREAD

**Issue by Hamburger6** [brave/brave-browser#10164]
## Steps to reproduce 

   1. 
   2. 
   3. 

## Issue happens on 
   - Current Play Store version?
   - Beta build?

## Device details
   - Install type (ARM, x86):
   - Device (Phone, Tablet, Phablet):
   - Android version:
   
### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by Hamburger6** [comment]
I'm using bravemonoarm64 on Android Alcatel V3 Android 9.0. problem with YouTube having ads still present every time I start a video. Can I use a different arm for an update?

**Comment by srirambv** [comment]
Installing different build doesn't solve the issue. It could be due to adblock list not being downloaded. Check if you see any `Update error` in `brave://components` for Adblock. If it is showing error try to update the component and see if it fixes the issue

**Comment by Hamburger6** [comment]
Just today I have downloaded last update to the browser monoarm64. Issue with YouTube ads still present every other video. Others have same issue? Not sure what to look at for the adblock since I never had the problem before. Ads are still blocked on other websites and never see any error in adblock.

**Comment by Hamburger6** [comment]
Still an issue with latest update. YouTube ads won't go away.

**Comment by Hamburger6** [comment]
Just now I searched on duckduckgo for components page and discovered some components were in need of updating. Is this going to be checked all the time since I saved the page as a bookmark.

**Comment by Hamburger6** [comment]
Components page updated but ads on YouTube.

**Comment by Hamburger6** [comment]
Components list doesn't always update and YouTube ads not blocked.

**Comment by Zireael07** [comment]
Having the issue with YT ads playing on desktop, too.

Wersja 1.7.90 Chromium: 80.0.3987.163 (Official Build) unknown (64-bit)

**Comment by antonok-edm** [comment]
Duplicate of https://github.com/brave/brave-browser/issues/3348 - this should be fixed now.
